go-controller/pkg/metrics/ovnkube_controller.go

```diff
@@ -269,6 +269,23 @@ var metricEgressFirewallCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "The number of egress firewall policies",
 })
 
+/** AdminNetworkPolicyMetrics Begin**/
+var metricANPCount = prometheus.NewGauge(prometheus.GaugeOpts{
+	Namespace: MetricOvnkubeNamespace,
+	Subsystem: MetricOvnkubeSubsystemController,
+	Name:      "num_anp",
```

Comment:
would the user know what `anp` means? 

Following this gold standard for naming conventions: https://prometheus.io/docs/practices/naming/

Id rename to `admin_network_policy_custom_resource_total`

Reply:
ah great point! thanks martin let me rename this.

Reply:
i guess i copied it form "num_egress_firewall" :dagger:

Reply:
changed this to what you suggested

Reply:
ya - totally understand - we have learned to align our naming with the industry standard since the mentioned metrics were created.